src/db/sqlite/lrsql/sqlite/sql/ddl.sql

```diff
@@ -426,3 +426,27 @@ ALTER TABLE activity_profile_document DROP COLUMN last_modified;
 -- :command :execute
 -- :doc Convert `activity_profile_document.last_modified` to timestamp - 04
 ALTER TABLE activity_profile_document RENAME COLUMN last_modified_tmp TO last_modified;
+
+--:name query-statement-to-actor-has-cascade-delete?
+select on_delete from pragma_foreign_key_list("statement_to_actor") where "table" ="xapi_statement"
```

Comment:
Any reason why the keywords in this query are suddenly lowercase?

Reply:
Also you missed a space after the `=` sign. Admittedly a very nitpicky comment.

Reply:
capitalized, added